src/main/java/strategy/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package strategy.operation;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        Integer oldSum = Storage.fruits.get(fruitTransaction.getFruit());
```

Comment:
Potential NullPointerException if the fruit is not present in the Storage.fruits map. Consider using the 'getOrDefault' method to handle this case.